tests/test_inifile.py

```diff
@@ -47,6 +48,11 @@ def test_description_file():
         "Sample description for test.\n"
     assert info['metadata']['description_content_type'] == 'text/x-rst'
 
+def test_missing_description_file():
+    with pytest.raises(ConfigError) as error:
```

Comment:
pytest shortcut:

```suggestion
    with pytest.raises(ConfigError, match=r"Description file .* does not exist"):
```

Reply:
Nice, I didn't know about that pytest feature.